backend/src/main/java/ai/verta/modeldb/comment/CommentServiceImpl.java

```diff
@@ -174,7 +253,19 @@ public void deleteExperimentRunComment(
       Boolean status =
           commentDAO.deleteComment(
               experimentRunEntity, request.getEntityId(), request.getId(), userInfo);
-      responseObserver.onNext(DeleteComment.Response.newBuilder().setStatus(status).build());
+      DeleteComment.Response response =
+          DeleteComment.Response.newBuilder().setStatus(status).build();
+      String projectId = experimentRunDAO.getProjectIdByExperimentRunId(request.getEntityId());
+      GetResourcesResponseItem responseItem =
+          roleService.getEntityResource(projectId, ModelDBServiceResourceTypes.PROJECT);
+      saveAuditLog(
+          Optional.of(userInfo),
+          ModelDBServiceActions.DELETE,
```

Comment:
Update. We're updating the ER, not deleting it.

Reply:
Resolved